### PR TITLE
feat: update load-gen example

### LINKED ISSUE #229: implement ellery's suggestions about using time-based constructs for loadgen, etc.
Make the load-gen example match the Python and Javascript examples.

- [x] logLevel
- [x] printStatsEveryNRequests -> showStatsInterval
  - [x] Takes a TimeSpan.
- [x]  totalNumberOfOperationsToExecute -> howLongToRun (?)
  - [x] Takes a TimeSpan.
- [x] Display maxRequestsPerSecond, numberOfConcurrentRequests, and howLongToRun to the user before running.

I skipped requestTimeout, the example uses the pre-built Laptop config.

These are very desirable UX improvements because with the current setup, if the user changes `maxRequestsPerSecond` or `numberOfConcurrentRequests`, it will have a big impact on how often (in wall clock time) the stats are printed, as well as how long the program will run for.  That's not intuitive for users.

### PR DESCRIPTION
**Pull request by schwern**
This is my first time doing dotnet; be savage.

Closes #229

### CODE REVIEW

File: examples/MomentoLoadGen/Program.cs [hunk 1]
```diff
@@ -407,45 +393,45 @@ static ILoggerFactory InitializeLogging()
...
+            CsharpLoadGeneratorOptions loadGeneratorOptions = new CsharpLoadGeneratorOptions(
+              ///
+              /// Controls the verbosity of the output during the run.
+              ///
+              logLevel: LogLevel.Debug,
+              ///
+              /// Each time the load generator has executed this many requests, it will
+              /// print out some statistics about throughput and latency.
+              ///
+              showStatsInterval: new TimeSpan(0, 0, 5),
```
Comment (cprice404) [inline examples/MomentoLoadGen/Program.cs:405]: one small nit - I believe the TimeSpan class has some static methods that are factory constructors that are something like "ofSeconds" or "ofMinutes".  I find those to be easier to read/review.

**Review by cprice404 — CHANGES_REQUESTED** [review]
this looks great other than one very small nit

File: examples/MomentoLoadGen/Program.cs (hunk 1 above)
Comment (schwern) [reply examples/MomentoLoadGen/Program.cs:405]: And I forgot to change the comment. 🙄

File: examples/MomentoLoadGen/Program.cs [hunk 2]
```diff
@@ -399,10 +399,11 @@ static async Task Main(string[] args)
               ///
               logLevel: LogLevel.Debug,
               ///
-              /// Each time the load generator has executed this many requests, it will
-              /// print out some statistics about throughput and latency.
+              /// Each time this amount of time has past, statistics about throughput and latency
```
Comment (cprice404) [inline examples/MomentoLoadGen/Program.cs:402]: nit: s/past/passed?

Comment (cprice404) [inline examples/MomentoLoadGen/Program.cs:402]:
```suggestion
              /// Each time this amount of time has passed, statistics about throughput and latency
```

**Review by cprice404 — APPROVED** [review]
:ty:

**Pull request merged**